### DISCUSSION TITLE
Sketch library bugs from preview/unified theme update

### DISCUSSION THREAD

**Discussion by elainelylee** [twilio-labs/paste#687] [Q&A] [answered]
Not sure if this is due to us being on different versions of Sketch and not being able to update our licenses at the moment, but my components are not syncing with the updated components. It's instead breaking my components.

For example, my button turned into this
![image](https://user-images.githubusercontent.com/6718764/91239048-4a86a180-e6f3-11ea-9e8d-9a3390f4133f.png)

So far, I mainly see it in the buttons components.

**Comment by serifluous** [comment]
@elainelylee Can you send us the Abstract link of the file you're seeing this issue in? We were seeing it in the Paste components libraries, too, and figured out a workaround but want to make sure it'll work for you.

**Reply by serifluous** [reply]
(got the link through Slack)

@elainelylee 

If you need a fix right away, what we did is delete all our instances of the Button, save and close the file, commit, and then re-place the symbols. It's not ideal (and I still saw the issue pop up later), but you can try it for now.

I'm attempting a fix on our end right now. To help troubleshoot:

1. Which button variants are you seeing it in? Is it just primary and secondary? Do you see it on the destructive button?
2. Is it only in the Console theme? If you place a "Default" (UDL) button symbol, do you also see the issue?

**Reply by elainelylee** [reply]
I'm seeing the issue in the required dot as well.

Originally, I saw it in the secondary button. When I restarted the app, it was happening to the primary buttons. Destructive buttons also have the issue. These are all Console versions. Base symbol parts for the buttons.

**Reply by serifluous** [reply]
Still trying to fix this. I think it might have to do with how the symbols are nested.

Can you share a screenshot of the required dot issue?

**Reply by elainelylee** [reply]
![image](https://user-images.githubusercontent.com/6718764/91246780-98a4a080-e705-11ea-8319-3f8a98dfa238.png)
I replaced the dot with the one from "Default" instead for now.

**Reply by serifluous** [reply]
@elainelylee Can you check the buttons and required indicator in your file to see if the issue's still there? I merged in a fix just now.

(For our own reference, what I did was replace the button background base symbols in the button state symbols)

**Reply by serifluous** [reply]
@kevhealy-git Let me know if your destructive button issue's been resolved, too, with the library update.

**Reply by elainelylee** [reply]
Primary button, Destructive button, and Required dot fixed on my end and working.

**Comment by elainelylee** [comment]
Along with the button issue, did our form fields and button size change (get smaller?). The modal doesn't seem to reflect this yet if the button size did change.

**Reply by serifluous** [reply]
Yep, our form fields and button sizes got smaller based on UDL updates.

The Alert, Card, and Modal symbols haven't been updated yet to reflect some of the UDL changes, but those will come soon.

**Comment by elainelylee** [comment, marked as answer]
Additional bugs that came with this library update:
Alignment of the text inside the select 
![image](https://user-images.githubusercontent.com/6718764/91333674-ba903880-e782-11ea-8f8a-963b3c4fe68e.png)

Unchecked checkbox got filled with a checkmark
![image](https://user-images.githubusercontent.com/6718764/91333730-d0056280-e782-11ea-9a74-c79d2022322a.png)

**Reply by serifluous** [reply]
Merged in a fix for the checkmark.

I can't duplicate the Select issue, though. Would you mind committing or sending a copy of the Sketch file so I can poke around in your file?

**Reply by elainelylee** [reply]
The checkmark is fixed on my end and works.

Alignment issue outstanding.

**Reply by serifluous** [reply]
@elainelylee Did the alignment issue get fixed with the newest version of Sketch?

**Reply by elainelylee** [reply]
Yes, this works. Looks like this 
![image](https://user-images.githubusercontent.com/6718764/94326190-54e8c500-ff57-11ea-9bfc-4ef65254360e.png)

**Comment by elainelylee** [comment]
![image](https://user-images.githubusercontent.com/6718764/91341644-a9e5bf80-e78e-11ea-859a-646e5e6ca21c.png)
Read Only input field looks like the default input field. Seems like the background color went missing.

**Reply by serifluous** [reply]
Merged in a fix for this, too.

**Reply by elainelylee** [reply]
this is fixed and works on my end

**Comment by elainelylee** [comment]
![image](https://user-images.githubusercontent.com/6718764/92046991-c9ee2380-ed38-11ea-9213-8441b0e635d5.png)

unselected state for error checkbox didn't get the update

**Reply by richbachman** [reply]
Hi @elainelylee. Thanks for catching this. I'll let the team know.

**Reply by kate-porter** [reply]
Noted! We'll get a fix for this merged in today and will let you know when it's ready for you.
